frontend/src/apis/analytics.ts

```diff
@@ -30,42 +31,47 @@ export const createAnalyticsDashboard = async (
 ) => {
   const result: any = await apiRequest(
     'post',
-    `/project/${dashboard.projectId}/dashboard`,
+    `/project/${dashboard.projectId}/${dashboard.appId}/dashboard`,
     dashboard
   );
   return result;
 };
 
 export const deleteAnalyticsDashboard = async (
   projectId: string,
+  appId: string,
   dashboardId: string
 ) => {
   const result: any = await apiRequest(
     'delete',
-    `/project/${projectId}/dashboard/${dashboardId}`
+    `/project/${projectId}/${appId}/dashboard/${dashboardId}`
   );
   return result;
 };
 
 export const getAnalyticsDashboard = async (
   projectId: string,
+  appId: string,
   dashboardId: string
 ) => {
   const result: any = await apiRequest(
     'get',
-    `/project/${projectId}/dashboard/${dashboardId}`
+    `/project/${projectId}/${appId}/dashboard/${dashboardId}`
   );
   return result;
 };
 
 export const getMetadataEventsList = async (params: {
   projectId: string;
   appId: string;
+  attribute?: boolean;
 }) => {
-  const result: any = await apiRequest(
-    'get',
-    `/metadata/events?projectId=${params.projectId}&appId=${params.appId}`
-  );
+  const { attribute } = params;
+  let path = `/metadata/events?projectId=${params.projectId}&appId=${params.appId}`;
```

Comment:
Why not define API path as `/metadata/${projectId}/${appId}/events` to make the consistent API sytle?

Reply:
track #215